src/core_plugins/kibana/public/management/sections/indices/add_data_steps/parse_csv_step/lib/__tests__/validate_headers.js

```diff
@@ -0,0 +1,68 @@
+import validateHeaders from '../validate_headers';
+import expect from 'expect.js';
+
+describe('validateHeaders', function () {
+
+  describe('simple validations', function () {
+
```

Comment:
Could you add a test for a non-array parameter? expect.js can run assertions on exceptions which will be handy